x-pack/plugins/security_solution/server/lib/detection_engine/rule_execution_log/rule_registry_log_client/rule_execution_field_map.ts

```diff
@@ -18,6 +18,9 @@ import {
  * @deprecated ruleExecutionFieldMap is kept here only as a reference. It will be superseded with EventLog implementation
  */
 export const ruleExecutionFieldMap = {
+  // [ALERT_OWNER]: { type: 'keyword', required: true },
```

Comment:
Should these be commented out still?

Reply:
These comments should actually be deleted. This was resolved here: https://github.com/elastic/kibana/pull/107581